Doesn't work with projects with spaces in the name

If the Unity project has spaces in the name, or the added csproj files contains spaces, it's unable to compile the projects.
For example, if I create a project with the default Unity name "New Unity Project", the resulting compile command line is "C:\Program Files\dotnet\dotnet.exe msbuild New Unity Project.Dependencies.msb4u.csproj -restore  -v:minimal -p:NuGetInteractive=true  -t:Build -p:Configuration=Release", which causes "MSBUILD : error MSB1008: Only one project can be specified.".
Adding quotes around the project name should solve this issue.

--- a/Source/MSBuildTools.Unity/Packages/com.microsoft.msbuildforunity/Editor/ProjectBuilder/MSBuildProjectBuilder.cs
+++ b/Source/MSBuildTools.Unity/Packages/com.microsoft.msbuildforunity/Editor/ProjectBuilder/MSBuildProjectBuilder.cs
@@ -396,7 +396,7 @@ namespace Microsoft.Build.Unity
 
         private static async Task<int> BuildProjectAsync(string projectPath, BuildEngine buildEngine, string arguments, IProgress<(string progressMessage, ProgressMessageType progressMessageType)> progress, CancellationToken cancellationToken)
         {
-            arguments = $"{Path.GetFileName(projectPath)} -restore {arguments}";
+            arguments = $"\"{Path.GetFileName(projectPath)}\" -restore {arguments}";
             string msBuildPath = null;
 
             switch (buildEngine)

